Uploaded my code for feature extraction

Astralknight532 opened the pull request:
This is the code that I have so far for extracting the features I was assigned. I have temporary placeholders in the section for handling skew and normalization because I'm unsure of whether these 4 features actually require it (especially OpenPorchSF & WoodDeckSF).

Commit:
Uploaded my code for feature extraction

This is the code that I have so far for extracting the features I was assigned. I have temporary placeholders in the section for handling skew and normalization because I'm unsure of whether these 4 features actually require it (especially OpenPorchSF & WoodDeckSF).

Commit:
Update kwok.py

Commit:
Update kwok.py

srwight commented on kwok.py:
```diff
@@ -0,0 +1,89 @@
+    # with the value used to filter the data or with some value close to it
+    for ga in myFeatures['GarageArea']: # Garage area in square feet
+        if (ga >= 1300):
+            myFeatures['GarageArea'].replace(to_replace = ga, value = 1300)
```
when you pick a seemingly arbitrary value, go ahead and explain why.  Where did the 1300 come from?

Same question below.

srwight commented on kwok.py:
```diff
@@ -0,0 +1,89 @@
+            myFeatures['WoodDeckSF'].replace(to_replace = wd, value = 790)
+            
+    # Handle skewness and/or normalization (the next few lines of code are placeholders)
+    #print("GarageArea skew: %s\n" % myFeatures['GarageArea'].skew())
```
Check with a few folks. Show them your graphs, and see what they think about the skew question. Then make a call and be prepared to explain it.

srwight reviewed (commented):
I have questions - see above. Also, make a decision on the skew. Talk to some folks and see what they think. I'm sure you'll come to a good decision.

Commit:
Added code to handle skew

There is now code to handle skew within the 4 features as well as fixes to some of the comments in the code.

disassembledd commented on kwok.py:
```diff
@@ -0,0 +1,103 @@
+    """
+    
+    # Picking out the assigned columns & putting them into a separate DataFrame
+    myFeatures = pd.DataFrame(dataset, columns = ['GarageArea', 'GarageCars', 'OpenPorchSF', 'WoodDeckSF'])
```
`myFeatures = dataset[['GarageArea', 'GarageCars', 'OpenPorchSF', 'WoodDeckSF']]`
This would be a more efficient way to grab the columns you're specifically looking for. You don't want to get into the habit of creating variables that take extra memory especially working with large data like we are. Keep in mind this is just me looking at the details, but these details are things that really matter on a large business scale.

srwight replied:
In fact, as written, it would throw an error. Consider changing to Alex's suggestion.

Astralknight532 replied:
I updated the code just now to have the suggested syntax.

Commit:
Update kwok.py

srwight reviewed (approved):
Much better
